display_information/display-annotation/src/main/java/com/esri/samples/display_annotation/DisplayAnnotationSample.java

```diff
@@ -65,8 +67,25 @@ public void start(Stage stage) {
             // add the annotation layer to the map
             map.getOperationalLayers().add(annotationLayer);
 
+            // add a done loading listener, with a runnable that gets triggered asynchronously when the feature layer has loaded
+            // check for the load status of the layer and if it hasn't loaded, report an error
+            riverFeatureLayer.addDoneLoadingListener(() -> {
+                if (riverFeatureLayer.getLoadStatus() != LoadStatus.LOADED) {
```

Comment:
This is okay, but seems a little overkill for a simple sample. We've had discussions before about omitting some best practices for the sake of brevity/focus. Should get approval from Lucas/Mike before making more of these changes.